pkg/util/downloader/downloader_test.go

```diff
@@ -13,6 +13,13 @@ var _ = Describe("Downloader", func() {
 	Context("Downloader test", func() {
 		var url = "https://github.com/devstream-io/devstream/releases/download/v0.0.1/argocdapp_0.0.1.so"
 		var targetDir = "tmp"
+
+		It("Should return err", func() {
```

Comment:
We can improve the name, for example:

It should return error if filename is empty and can't be parsed from the url.

Reply:
Please pay attention here. @summingyu